src/main/java/com/rultor/profiles/Profiles.java

```diff
@@ -92,17 +157,26 @@ private static Profile fetch(final XML xml) {
                 new TalkIssues(Profiles.github(), xml).get().repo()
             );
         } else {
-            profile = new GithubProfile(
-                Profiles.github().repos().get(
-                    new Coordinates.Simple(
+            final Profile master = new GithubProfile(
+                    new TalkIssues(Profiles.github(), xml).get().repo()
+            );
+            profile = this.validated(
+                master,
+                this.merged(
+                    master,
+                    new GithubProfile(
+                        Profiles.github().repos().get(
+                            new Coordinates.Simple(
```

Comment:
@xupyprmv just pass this String here:
```
                               xml.xpath(
                                    "//request/args/arg[@name='fork']/text()"
                                ).get(0)
```

 instead of the actual fetched repo ;) instantly saves us one call to the GitHub API here, Rultor will eventually have to fetch the fork contents anyhow, no need to know those here already imo :)